config/sidebar.yml

```diff
@@ -307,8 +307,14 @@ articles:
       - title: "Overview"
         url: "/universal-login"
 
-      - title: "Password Reset Page"
```

Comment:
Did you mean to delete the Password Reset Page entry?

Reply:
No, I just put it back